dataland-e2etests/src/test/kotlin/org/dataland/e2etests/EuTaxonomyControllerTest.kt

```diff
@@ -0,0 +1,78 @@
+package org.dataland.e2etests
+
+import org.dataland.datalandbackend.openApiClient.api.CompanyDataControllerApi
+import org.dataland.datalandbackend.openApiClient.api.EuTaxonomyDataControllerApi
+import org.dataland.datalandbackend.openApiClient.model.CompaniesRequestBody
+import org.dataland.datalandbackend.openApiClient.model.DataIdentifier
+import org.dataland.datalandbackend.openApiClient.model.DataSetMetaInformation
+import org.dataland.datalandbackend.openApiClient.model.EuTaxonomyData
+import org.dataland.datalandbackend.openApiClient.model.EuTaxonomyDataSet
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Assertions.assertTrue
+import org.junit.jupiter.api.Test
+import java.math.BigDecimal
+
+class EuTaxonomyControllerTest {
+    val companyDataControllerApi = CompanyDataControllerApi(basePath = "http://proxy:80/api")
+    val euTaxonomyDataControllerApi = EuTaxonomyDataControllerApi(basePath = "http://proxy:80/api")
+    val testEuTaxonomyDataSet = EuTaxonomyDataSet(
```

Comment:
Das ist die gleiche Funktion wie in CompanyDataControllerTest.kt. 

Können wir das vereinheitlichen?

Reply:
Wie wäre es mit einer DataCreator Klasse im e2e-Test, deren Methoden sprechend heißen und die in den Tests selbst dann einfach nur benutzt werden (z.B. `val testEuDataSet = DummyDataGenerator().createRandonEuTaxonomyDataSet()`).

Z.B. auch als randomized Data Creator, der TestDaten zufällig generiert.

Habe es mal nicht-randomized umgesetzt und gepusht.

Reply:
ok